imports/ui/Index.js

```diff
@@ -1,9 +1,10 @@
-import './About.html';
-import './Home.html';
+// Orphaned html files
```

Comment:
Intent of this comment?

Reply:
Copied from before, I'll just delete it.